uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/jpa/PersonAttributesGroupTestDefinitionImpl.java

```diff
@@ -46,14 +51,14 @@
  * @author Shawn Connolly, sconnolly@unicon.net
  */
 @Entity
-@Table(name = "UP_PAGS_TEST")
+@Table(name = JpaPersonAttributesGroupTestDefinitionDao.TABLENAME_PREFIX)
```

Comment:
Is it weird that the name of the table would be the TABLENAME  **PREFIX**  (instead of being the TABLENAME ?)